front/components/assistant/AssistantDetailsDropdownMenu.tsx

```diff
@@ -126,81 +129,109 @@ export function AssistantDetailsDropdownMenu({
 
   return (
     <>
-      {canDelete && showDeletionModal && (
+      {allowDeletion && showDeletionModal && (
         <DeleteAssistantDialog
           owner={owner}
-          agentConfigurationId={showDeletionModal.sId}
+          agentConfiguration={showDeletionModal}
           show={!!showDeletionModal}
           onClose={() => setShowDeletionModal(null)}
-          onDelete={onAgentDeletion}
           isPrivateAssistant={showDeletionModal.scope === "private"}
         />
       )}
 
       <DropdownMenu className="text-element-700">
-        <DropdownMenu.Button>{dropdownButton}</DropdownMenu.Button>
-        <DropdownMenu.Items width={220}>
-          <DropdownMenu.Item
-            label="Start conversation"
-            link={{
-              href: `/w/${owner.sId}/assistant/new?assistant=${agentConfiguration.sId}`,
-            }}
-            icon={ChatBubbleBottomCenterTextIcon}
-          />
-          {!isGlobalAgent && (
-            <>
-              <DropdownMenu.SectionHeader label="Edition" />
-
-              {/* Should use the router to have a better navigation experience */}
-              {(isBuilder(owner) || !isAgentWorkspace) && (
-                <DropdownMenu.Item
-                  label="Edit"
-                  link={{
-                    href: `/w/${owner.sId}/builder/assistants/${
-                      agentConfiguration.sId
-                    }?flow=${
-                      isAgentWorkspace
-                        ? "workspace_assistants"
-                        : "personal_assistants"
-                    }`,
-                  }}
-                  icon={PencilSquareIcon}
-                />
-              )}
+        {({ close }) => (
+          <>
+            <DropdownMenu.Button>{dropdownButton}</DropdownMenu.Button>
+            <DropdownMenu.Items width={230}>
```

Comment:
Could we get rid of the "width" parameter?

Reply:
In general, probably, in particular, I think it might be a time-sink :(
(i'll try to remove and see how it behave)

Reply:
We usually kinda try to fix the sparkle component itself. But of course if it's to spend a day on it, let's just go with the `width` parameter 👍🏼 
Maybe let's add a TODO as well?

Reply:
Yep :)